index.html

```diff
@@ -1,112 +1,70 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no">
-
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>shape of my code</title>
     <link rel="stylesheet" href="style.css" type="text/css">
-    <link href="https://fonts.googleapis.com/css2?family=Nunito:ital,wght@0,400;1,200;1,700;1,900&display=swap" rel="stylesheet">
+</head>
+<body>
+    <div class="container-half"></div>
+    <div class="container-half" id="c-h-right"></div>
+        <div class="bg-modal">
+            <div class="modal-content">
+                <form action="">
+                    <label for="nickname">Who are you?</label>
+                        <input type="text" class="input-box" id="nickname" placeholder="Nickname">
+                    <label for="amount">How many shapes you want?</label>
+                        <input type="number" class="input-box" id="amount" placeholder="Amount">
+                    <input type="radio" class="input-radio" name="shape" value="0"> Triangle<br>
+                    <input type="radio" class="input-radio" name="shape" value="1"> Square<br>
+                    <input type="radio" class="input-radio" name="shape" value="2"> Circle<br>
+                    <button type="submit" class="btn-submit" disabled>OK</button>
+                    <button type="reset" class="btn-cancel">CANCEL</button>
+                </form>
+            </div>
+        </div>
+<script>
+    window.onload = function() {
+        document.querySelector('.bg-modal').style.display = 'flex';
+    }
+        document.querySelector('.btn-cancel').addEventListener('click', function() {
+            document.querySelector('.bg-modal').style.display = 'none';

```

Comment:
zauwazylem, ze .bg-modal pojawia sie u Ciebie kilka razy. Moze warto wrzucic to do stalej? pozniej bedziesz zmienial tylko w jednym miejscu. Mam na mysli cos takiego:

const MODAL_CLASS = '.bg-modal';

document.querySelector(MODAL_CLASS)...